code/cli/src/operational.rs

```diff
@@ -0,0 +1,738 @@
+	"5wtELdwzqyuWCR78W1jDyQ8nb1jViGcgVZn4BqQbGYU7gCEi",
+	"5CJUn2syLhz6nJVdLpxNSxkoBYyGHWurE8UxYKZ5obx4FXH4",
+	"5x27jpexjzR1G5cM4aSb9iZH7CskQ4n2ocVkR3o16Wkarsbu",
+	"5vFtm6WHwPGsCSunvmxN6FqReV1bcWw75FD7KUiZvV7vsy8t",
+	"5x79GYLQNxcmryb8jRQkxkWs9LSTzKwPn32NiQd6WXsuDrb4",
+	"5v8VpcLqouh25Q6cLM4CTvHfYT5XqNNiQBGVtcS768Cd59RX",
+];
+
+const TECH_COUNCIL: &[&str] = &[
+	"5GQjasGFwLDnyTcBvvUwRA1smNU8JN15MPYx2TV8N62iopw2",
+	"5GF8HUxZ1tCHWVZMoGVqDWe8EtdnKznHEbaguRQNqvMkw34H",
+	"5uVVWApbzcdgrkfYDRSn42Zcs3GnT9ypU7EHnzCuEsLyztm3",
+	"5FCCuxH6VuMdCdsqiYAHg515FJopNvSYVdQCHGBVunAdKh3M",
+	"5HY3FuywmNdPfVkG53ZkSruJcLLxJN51Skei8rqYWFFhAu2r",
+	"5wXghvLSiSksRytoyGPq9mQ12CsfcP49tng3XSMJYEYgcWZr",
+	"5EyK95UCZaWbD3YUvvQKgaSYeP9L6rhd6ddi25xLt3crPCBX",
+];
+
+const TREASURY_MSIG: &[&str] = &[
+	"5uuVmG72PzKfYNewVUfVmVxQMRXhjhB6cJnzLhqNCxVu3dGp",
+	"5C5RYpMfwqbJTbTCtsG3MUsuD7782Xv2LhohjVBGzG5iucuL",
+	"5urB7uAgHFdgFRuohhQNHMdGHP4uRrjaKVnRUyMk3RP9bJpt",
+	"5uQ6tWbxJvoS84QE8XrM9QApiwsxXYmjfXm28DaoAiDCj6eF",
+	"5uJueaCukS1qFXPTeXycCBd3SVbumV38i3mpbG8gPkAAip7r",
+	"5v2R5BCdBfyWJTikPB1zdcBB2W73Ut4DVPTvsraAkqsL9bkS",
+	"5CqZ9mELKwk7hV6WKVzm3M5GnXwaibFCh714WeW54aFey88E",
+	"5u7x85XYQ22Yxj6XZHcr6kQk2xhd2i7fNECAVRQpGmbNK8dJ",
+	"5x29WEJkvixjzPqAwXMSYatyZ1DpjHcT6DYLJkTyXwmENoks",
+	"5weoaaztdvn88s6Cn23zahxe8VCfQ3UsKJeA57LM4j5WJdjX",
+	"5z95uz5LfGHFSukyEBFezeebd278UmLvKovi8hfoJxcwRdaw",
+];
+
+const ECOSYSTEM_INCENTIVES_MSIG: &[&str] = &[
+	"5ykL57L8KLrUn5mGKg23WHtQAuBada2E5rtuoNv1NoTSBnP2",
+	"5HN4nzgzCWqT5oqmycpADH8efNxJsywYuTJExbyPrcSrfLhu",
+	"5xFwuUQsMnkG92xgcLQ2XyvGpZjq7bGTevSqZCbMAD7yHUjF",
+	"5wWsrkiWhmMiy7DzjurYKSQ9zbEohvZPLEqhHVuqpwAYrBXZ",
+	"5uQjawk289JaAjK3cQJaBzQ31hdSdSNQAUD1qVc2UkzEyAVE",
+	"5vdHj3dgwAeUUSyg89MjxU3ksuPNthhkENDVYtQAbQLRMetX",
+	"5HQaiGrnau6NqvXFikzZ9pz9TpceY4gXiyprZCYVtvbMwhBo",
+	"5zEQ4ourpwXdtn1s5Pm3hEZcfE9wqcQXV1WM2DvajLcRZaUu",
+	"5z4feDAoLtsBTcF9a72uauYtkdU6y3WNYLtBN8det3kdeX8b",
+	"5wCX2QU7wTd4wMHysQF6LQLJ2yynrgvmEpsUAJeYTt3KkVjM",
+	"5vhwKyotaohcRQUp9Lr8wUgUBwfaDqZZYSXu31sQQ3n7Yvaf",
+];
+
+const LIQUIDITY_PROGRAMS_MSIG: &[&str] = &[
+	"5yPPBRbLdyGZX79pP2iHFdgzjEjmCY29Tm8qB6PebyWGdpSX",
+	"5FsBvfR6A1otoZMGYHRGJ4UHPo1nRrNuFhrBZ84FbKK8o8mU",
+	"5wwB1mdc7HTuedtcSrZ8yvP5715Nb7FKdojHMr776gGc5ipQ",
+	"5vm73wPXjREjoSVA3ycfNLbjMPJkFLfc7hHjV7AqXzqKLaX8",
+	"5uV7onK9rBbo17yQjy833URFMtAri6N8ieb3eocyGZjRLmS9",
+	"5xpsDMcor2rmkDVPfzJ3uDUBjEo7Wf5C38Kv3Cje3iTfq9ZX",
+	"5ENfibEkoTnkPnmWCf3KpWTkgMh9JEieVkTiwSRxL4bEbQmr",
+	"5zEBXadXr7WEgo32rSxkzkzN4VKvqqZs8Xmv9aM8CMsPns9F",
+	"5wA6tsLsdoZ4jxkTXWSj6zsCzNqutMDQ9RnhkqgG6p41kAcf",
+	"5z7XXgsGjkmpN3Yd8kazWkTFmFFd7yPMF8M7qynvNT1YE8FC",
+	"5xnXawcszffzPjFtEWiCPsgqVKRfdkZa2vciePhjpEUwUkdT",
+];
+
+const TEAM_MSIG: &[&str] = &[
+	"5wFGWTb9pxFXbFjKyST2JSmVLz7dHVy6bKdxEGh3Y5iCa5v8",
+	"5uiQYPKrG72RRikpvJfuQE8ghKB47rKxnEuiymCywzsKRUMb",
+	"5yX8dWEBdWivoPh7tkxoby2wkD78sgbm3Bk2Rx4jsGugwXdN",
+	"5vJwkhJCnpjq2R9isLnnSAXn3HBVVWRkvG5SabM5XaUGYrb1",
+	"5E9sbF2Ty4eq6oE69kX7EVBNc4CFZMzYn7B9gyyHc9natSEu",
+];
```

Comment:
I have no idea if all of these addresses are correct. But I assume that you've double checked them prior to opening the PR for review

Reply:
Will also be reviewed with me and Omar tonight

Reply:
They are sir